### ISSUE TITLE
FR: Ollama Chat API

### ISSUE THREAD

**Issue by emsi** [promptfoo/promptfoo#340]
# Feature Request

## Ollama Chat API

Starting v0.1.14 Ollama implemented chat API (akin to OpenAI's):
https://github.com/jmorganca/ollama/releases/tag/v0.1.14

```
curl http://localhost:11434/api/chat -d '{
  "model": "mistral",
  "messages": [
    { "role": "system", "content": "You are a helpful assistant that answers concisely." },
    { "role": "user", "content": "why is the sky blue?" }
  ]
}'
```

This allows for more  elaborate prompting techniques, like example answers, etc. without using raw prompt templates.
It'd be great if promptfoo implements that API provider.

**Comment by typpo** [comment]
Thanks for pointing out this new API.  This implementation is in progress: PR #342

**Comment by typpo** [comment]
Resolved in version 0.33.0
